src/Columns/ColumnAggregateFunction.cpp

```diff
@@ -246,6 +265,10 @@ void ColumnAggregateFunction::ensureOwnership()
                 for (size_t i = 0; i < rollback_pos; ++i)
                     func->destroy(data[i]);
 
+            /// Restore original data
+            const ColumnAggregateFunction & from = *assert_cast<const ColumnAggregateFunction *>(src.get());
+            memcpy(data.data(), &from.data, rollback_pos * sizeof(data[0]));  // NOLINT(bugprone-bitwise-pointer-cast)
```

Comment:
This is wrong, and it is not possible to use source column, since we can copy multiple times from it, and even saving offset in src and offset in data is not enough, need to save old data